lisp/ess-custom.el

```diff
@@ -686,8 +686,8 @@ If a list of the form '(N) where N is a number, the statements
 are indented at the previous line indentation + N characters:
 
   a <- some.function(other.function(
-     arg1,
```

Comment:
I thought it was a documentation mistake. The original example had indents of 3 spaces whereas 2 spaces are used in the rest of the examples.

Or am I missing something?